Source/WebCore/platform/graphics/gstreamer/mse/AppendPipeline.cpp

```diff
@@ -107,7 +106,7 @@ AppendPipeline::AppendPipeline(SourceBufferPrivateGStreamer& sourceBufferPrivate
     ASSERT(isMainThread());
     std::call_once(s_staticInitializationFlag, AppendPipeline::staticInitialization);
 
-    GST_TRACE("Creating AppendPipeline (%p)", this);
+    GST_TRACE_OBJECT(m_playerPrivate->pipeline(), "Creating AppendPipeline (%p)", this);
```

Comment:
GST_DEBUG_OBJECT -- AppendPipeline creation is infrequent enough to not warrant TRACE.